src/util.c

```diff
@@ -100,10 +105,34 @@ static void hash2hex(const uint8_t *hash, char *dst)
 	bin2hex(hash, MD5_DIGEST_LENGTH, dst, 16*2 + 1);
 }
 
-void pg_md5_encrypt(const char *part1,
+bool pg_md5_encrypt(const char *part1,
 		    const char *part2, size_t part2len,
 		    char *dest)
 {
+	bool result = false;
+#ifdef USUAL_LIBSSL_FOR_TLS
+	EVP_MD_CTX *mdctx;
+	uint8_t hash[MD5_DIGEST_LENGTH];
+
+	ERR_clear_error();
+	mdctx = EVP_MD_CTX_create();
```

Comment:
It might be nice to do this allocation only once for the lifetime of the program, instead of for every call to pg_md5_encrypt.

Reply:
To be clear, I don't think this is strictly necessary for this PR.